lib/Dialect/Calyx/CalyxOps.cpp

```diff
@@ -267,11 +267,33 @@ static LogicalResult verifyComponentOp(ComponentOp op) {
     else if (isa<ControlOp>(bodyOp))
       ++numControl;
   }
-  if (numWires == 1 && numControl == 1)
-    return success();
+  if (!(numWires == 1 && numControl == 1))
```

Comment:
style nit: i think it reads more clearly if you apply demorgan's law to this condition. up to you though.

Reply:
Changed. Thanks for the review.